pyrate/core/phase_closure/closure_check.py

```diff
@@ -0,0 +1,203 @@
+#   This Python module is part of the PyRate software package.
+#
+#   Copyright 2021 Geoscience Australia
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+from collections import defaultdict
+from typing import List, Dict, Tuple
+import numpy as np
+from pyrate.core import config as cf, mpiops
+from pyrate.core.phase_closure.mst_closure import find_signed_closed_loops, sort_loops_based_on_weights_and_date, \
+    WeightedLoop, Edge
+from pyrate.configuration import Configuration
+from pyrate.core.phase_closure.sum_closure import sum_phase_closures
+from pyrate.core.phase_closure.plot_closure import plot_closure
+from pyrate.core.shared import Ifg
+from pyrate.core.logger import pyratelogger as log
+
+
+def detect_ps_with_unwrapping_errors(check_ps: np.ndarray, num_occurences_each_ifg: np.ndarray, params: dict) \
+        -> np.ndarray:
+    """
+    find where in the phase data exceed the PHASE_UNWRAP_ERROR_THRESHOLD, and assign nans to those pixels in all ifgs
+    """
```

Comment:
This is just one example (applicable universally to all funcs). What is missing here is a description for each variable. i.e. what are `check_ps`, `num_occurences_each_ifg`, `params`?

Reply:
Docstrings have been improved in 0d9a378, e4cf79c and e7a2aa6